<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <title>Artificial Intelligence for Protein Design, PKU Summer School 2025</title>
  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <link rel="icon" type="image/png" href="/images/favicon-32x32.svg">
  <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;700&display=swap" rel="stylesheet">
  <link href="https://deepgraphlearning.github.io/ProteinTutorial_AAAI2025/assets/css/style.css" rel="stylesheet">
  
  <meta property="og:title" content="" />
  <meta property="og:type" content="" />
  <meta property="og:url" content="" />
  <meta property="og:image" content="" />
  <meta property="og:description" content="" />
</head>
<body class='page '>
  <div id="main-menu-mobile" class="main-menu-mobile">
  <ul>
    
  </ul>
</div>

  <div id="wrapper" class="wrapper">
    
    <div class="container pb-6 pt-6 pt-md-10 pb-md-10">
  <div class="row justify-content-start">
    <div class="col-12 col-md-12">
      <h1 class="title">PKU Summer School 2025<br>Artificial Intelligence for Protein Design</h1>
      <h5 style="font-family: Playfair Display;">28 July - 1 August, 9am - 12 am</h5>
      
<div class="row">

  <div class="col-12 col-md-4 mb-2">
    <div class="team team-summary team-summary-large">
      
      <div class="team-image">
        <img alt="Jian Tang" class="img-fluid mb-2" src="assets/img/jian_tang.webp" />
      </div>
      
      <div class="team-meta">
        <h2 class="team-name"><a href="https://jian-tang.com/">Jian Tang</a></h2>
        <p class="team-description">MILA & BioGeometry</p>
      </div>
    </div>
  </div>

  <div class="col-12 col-md-4 mb-2">
    <div class="team team-summary team-summary-large">
      
      <div class="team-image">
        <img alt="Chunfu Xu" class="img-fluid mb-2" src="assets/img/chunfu_xu.jpg" />
      </div>
      
      <div class="team-meta">
        <h2 class="team-name"><a href="http://www.nibs.ac.cn/yjsjyimgshow.php?cid=5&sid=6&id=2467/">Chunfu Xu</a></h2>
        <p class="team-description">NIBS & Tsinghua</p>
      </div>
    </div>
  </div>

  <div class="col-12 col-md-4 mb-2">
    <div class="team team-summary team-summary-large">
      
      <div class="team-image">
        <img alt="Zhen Liu" class="img-fluid mb-2" src="assets/img/zhen_liu.jpg" />
      </div>
      
      <div class="team-meta">
        <h2 class="team-name"><a href="http://web.liuzhenlab.com/">Zhen Liu</a></h2>
        <p class="team-description">NIBS & Tsinghua</p>
      </div>
    </div>
  </div>
  
</div>

<h2 id="instructors">Instructors</h2>

<ul>
  <li><strong>Jian Tang</strong>, Associate Professor, Mila-Quebec AI Institute (Mila) & Founder, BioGeometry, <a href="https://jian-tang.com/">website</a><br />
    <ul>
      <li>Jian Tang is an associate professor at Mila, the leading AI institute founded by AI pioneer and Turing Award Laureate Yoshua Bengio, and a CIFAR AI Research Chair. His research focuses on deep generative models, geometric deep learning, and their applications to protein design. He is also the founder of BioGeometry, a leading AI startup focusing on developing generative AI for protein design.
      </li>
    </ul>
  </li>
  <li><strong>Chunfu Xu</strong>, Assistant Professor,  National Institute of Biological Sciences (NIBS) & Tsinghua University, <a href="http://www.nibs.ac.cn/yjsjyimgshow.php?cid=5&sid=6&id=2467/">website</a><br />
    <ul>
      <li>
        Chunfu Xu is an assistant investigator at the National Institute of Biological Sciences (NIBS) and an assistant professor at Tsinghua University. He received his Ph.D. from Emory University, and later joined Prof. David Bakerâ€™s lab for postdoctoral research. He established his own lab at NIBS in 2022, focusing on developing deep learning-based methods for protein design and exploring their applications in creating functional proteins to address challenges in basic science, biotechnology, and medicine.
      </li>
    </ul>
  </li>
  <li><strong>Zhen Liu</strong>, Assistant Professor,  National Institute of Biological Sciences (NIBS) & Tsinghua University, <a href="http://web.liuzhenlab.com/">website</a><br />
    <ul>
      <li>Dr. Liu is a Principal Investigator at the National Institute of Biological Sciences (NIBS). He earned his Ph.D. in Chemistry from The Scripps Research Institute (TSRI). In 2019, he served as a Postdoctoral Fellow at Caltech in the laboratory of Prof. Frances Arnold, who received the 2018 Nobel Prize in Chemistry. Dr. Liu started his independent career in 2022, focusing on biocatalysis and the development of novel enzyme-engineering strategies.
      </li>
    </ul>
  </li>
</ul>

<h2 id="introduction">Course Introduction</h2>
<p>Proteins are the fundamental workhorses of living cells, playing a pivotal role in carrying out nearly all cellular functions. Understanding protein function and designing de novo proteins is crucial for a broad range of applications across various industries, including biomedicine, environmental sustainability, agriculture, cosmetics, materials science, and food production. In recent years, advances in artificial intelligence have opened new frontiers in protein design (e.g., AlphaFold2, AlphaFold3, RFDiffusion), enabling the development of novel proteins with tailored functionalities. This course aims to give an introduction to AI for protein design, the latest progress, as well as their applications in real-world problems. We equip students with the tools to leverage cutting-edge machine learning models for protein structure prediction, function annotation, and the design of synthetic proteins. Through hands-on learning and real-world case studies, students will gain expertise in applying AI techniques to solve complex challenges in protein design, ultimately advancing innovation across multiple sectors.
<br>
<br>
Evaluation: 10% participation + 90% course project
</p>

<!-- <h2 id="schedule">Schedule</h2>
<p>8:30 am - 12:30 pm EST, February 26, 2025</p>

<h2 id="schedule">Location</h2>
<p>Room 117, Philadelphia Convention Center, Philadelphia, PA USA</p>

<h2 id="slides">Slides</h2>
<p>The slides can be found <a href="TBA">here</a>.</p> -->

<h2 id="outline">Outline</h2>

<ul>
  <li><strong>Day 1: Introduction to AI, Proteins, and Computational Protein Design [Jian, Chunfu]</strong>
    <ul>
      <li>Introduction to AI, Deep Learning, Transformers, Graph Neural Networks [Jian]
      </li>
      <li>Introduction to Proteins and Computational Protein Design [Chunfu]
      </li>
    </ul>
  </li>
  <li><strong>Day 2: Protein Representation Learning and Protein Structure Prediction [Jian]</strong>
    <ul>
      <li>Protein Representation Learning, Function Prediction
      </li>
      <li>Protein Structure and Dynamics Prediction
      </li>
    </ul>
  </li>
  <li><strong>Day 3: Generative Models and Protein Design [Jian]</strong>
    <ul>
      <li>Autoregressive Models, Diffusion Models
      </li>
      <li>Protein Generative Models
      </li>
    </ul>
  </li>
  <li><strong>Day 4: Basic Strategies and Applications of Enzyme Catalysis [Zhen Liu]</strong>
    <ul>
      <li>Basic strategies and applications of biocatalysis
      </li>
      <li>New-to-nature biocatalysis and novel protein engineering strategies
      </li>
    </ul>
  </li>
  <li><strong>Day 5: Applications in Antibody Design and Other Proteins [Chunfu, Jian]</strong>
  </li>


</ul>

<h2 id="references">References</h2>

<h3>Introduction to Deep Learning</h3>
<ol class="bibliography">
<li><span id="goodfellow2016deep">Goodfellow, Ian, et al. Deep learning. Vol. 1. No. 2. Cambridge: MIT press, 2016.</span></li>
<li><span id="vaswani2017attention">Vaswani, Ashish, et al. "Attention is all you need." Advances in neural information processing systems 30 (2017).</span></li>
<li><span id="hamilton2020graph">Hamilton, William L. Graph representation learning. Morgan & Claypool Publishers, 2020.</span></li>
</ol>

<h3>Representation Learning and Protein Structure Prediction</h3>
<ol class="bibliography">
<li><span id="madani2023progen">Madani, Ali, et al. "Large language models generate functional protein sequences across diverse families." <i>Nature Biotechnology 2023</i>.</span></li>
<li><span id="rives2021esm">Rives, Alexander, et al. "Biological structure and function emerge from scaling unsupervised learning to 250 million protein sequences." <i>PNAS 2021</i>.</span></li>
<li><span id="lin2023esm2">Lin, Zeming, et al. "Evolutionary-scale prediction of atomic-level protein structure with a language model." <i>Science 2023</i>.</span></li>
<li><span id="elnaggar2021prottrans">Elnaggar, Ahmed, et al. "Prottrans: Toward understanding the language of life through self-supervised learning." <i>TPAMI 2021</i>.</span></li>
<li><span id="zhang2023esmgearnet">Zhang, Zuobai, et al. "A systematic study of joint representation learning on protein sequences and structures." <i>ArXiv Preprint ArXiv:2303.06275</i>.</span></li>
<li><span id="hayes2024esm3">Hayes, Thomas, et al. "Simulating 500 million years of evolution with a language model." <i>Science 2024</i>.</span></li>
<li><span id="wang2024dplm">Wang, Xinyou, et al. "Diffusion Language Models Are Versatile Protein Learners." <i>ICML 2024</i>.</span></li>
<li><span id="satorras2021egnn">Satorras, Victor Garcia, Emiel Hoogeboom, and Max Welling. "E(n) equivariant graph neural networks." <i>ICML 2021</i>.</span></li>
<li><span id="jing2021gvp">Jing, Bowen, et al. "Learning from protein structure with geometric vector perceptrons." <i>ICLR 2021</i>.</span></li>
<li><span id="zhang2023gearnet">Zhang, Zuobai, et al. "Protein representation learning by geometric structure pretraining." <i>ICLR 2023</i>.</span></li>
<li><span id="fan2023cdconv">Fan, Hehe, et al. "Continuous-discrete convolution for geometry-sequence modeling in proteins." <i>ICLR 2023</i>.</span></li>
<li><span id="su2023saprot">Su, Jin, et al. "Saprot: Protein language modeling with structure-aware vocabulary." <i>ICLR 2024</i>.</span></li>
<li><span id="wang2024dplm2">Wang, Xinyou, et al. "DPLM-2: A multimodal diffusion protein language model." <i>ICLR 2025</i>.</span></li>
<li><span id="notin2023proteingym">Notin, Pascal, et al. "ProteinGym: Large-Scale Benchmarks for Protein Design and Fitness Prediction." <i>NeurIPS 2023</i>.</span></li>
<li><span id="cai2024gearbind">Cai, Huiyu, et al. "Pretrainable Geometric Graph Neural Network for Antibody Affinity Maturation." <i>Nature Communications 2024</i>.</span></li>
<li><span id="shan2022bindddg">Shan, Sisi, et al. "Deep Learning-Guided Optimization of Human Antibody Against SARS-CoV-2 Variants with Broad Neutralization." <i>PNAS 2022</i>.</span></li>
</ol>

<h3>Generative Models, Protein Design</h3>
<ol class="bibliography">
<li><span id="hsu2022esmif">Hsu, Chloe, et al. "Learning inverse folding from millions of predicted structures." <i>ICLR 2022.</i></span></li>
<li><span id="dauparas2022proteinmpnn">Dauparas, Justas, et al. "Robust deep learning-based protein sequence design using ProteinMPNN." <i>Science 2022</i>.</span></li>
<li><span id="yim2023framediff">Yim, Jason, et al. "SE (3) diffusion model with application to protein backbone generation." <i>ICML 2023</i>.</span></li>
<li><span id="yim2023frameflow">Yim, Jason, et al. "Fast protein backbone generation with SE (3) flow matching." arXiv preprint <i>ArXiv Preprint ArXiv:2310.05297</i>.</span></li>
<li><span id="lin2023genie">Lin, Yeqing, and Mohammed Alquraishi. "Generating Novel, Designable, and Diverse Protein Structures by Equivariantly Diffusing Oriented Residue Clouds." <i>ICML 2023</i>.</span></li>
<li><span id="lin2024genie2">Lin, Yeqing, et al. "Out of Many, One: Designing and Scaffolding Proteins at the Scale of the Structural Universe with Genie 2." <i>ArXiv Preprint ArXiv:2405.15489</i>.</span></li>
<li><span id="ingraham2023chroma">Ingraham, John B., et al. "Illuminating protein space with a programmable generative model." <i>Nature 2023</i>.</span></li>
<li><span id="watson2023rfdiffusion">Watson, Joseph L., et al. "De novo design of protein structure and function with RFdiffusion." <i>Nature 2023</i>.</span></li>
<li><span id="bose2023foldflow">Bose, Joey, et al. "SE (3)-Stochastic Flow Matching for Protein Backbone Generation." <i>ICLR 2024</i>.</span></li>
<li><span id="huguet2024foldflow2">Huguet, Guillaume, et al. "Sequence-Augmented SE (3)-Flow Matching For Conditional Protein Backbone Generation." <i>NeurIPS 2024</i>.</span></li>
<li><span id="shi2023protseed">Shi, Chence, et al. "Protein Sequence and Structure Co-Design with Equivariant Translation." <i>ICLR 2023</i>.</span></li>
<li><span id="lisanza2024proteingenerator">Lisanza, Sidney Lyayuga, et al. "Joint generation of protein sequence and structure with RoseTTAFold sequence space diffusion." <i>Nature Biotechnology 2024</i>.</span></li>
<li><span id="campbell2024multiflow">Campbell, Andrew, et al. "Generative Flows on Discrete State-Spaces: Enabling Multimodal Flows with Applications to Protein Co-Design." <i>ICML 2024</i>.</span></li>
<li><span id="chu2024protpardelle">Chu, Alexander E., et al. "An all-atom protein generative model." <i>PNAS 2024</i>.</span></li>
</ol>

<h3>Applications of Enzyme Catalysis</h3>
<ol class="bibliography">
<li><span id="arnold2017directed">Arnold, Frances H. "Directed evolution: bringing new chemistry to life." Angewandte Chemie (International Ed. in English) 57.16 (2017): 4143. <a href="https://onlinelibrary.wiley.com/doi/epdf/10.1002/anie.201708408" target="_blank">link</a></span></li>
<li><span id="bell2021biocatalysis">Bell, Elizabeth L., et al. "Biocatalysis." Nature Reviews Methods Primers 1.1 (2021): 46. <a href="https://www.nature.com/articles/s43586-021-00044-z#citeas" target="_blank">link</a></span></li>
<li><span id="savile2010biocatalytic">Savile, Christopher K., et al. "Biocatalytic asymmetric synthesis of chiral amines from ketones applied to sitagliptin manufacture." Science 329.5989 (2010): 305-309. <a href="https://www.science.org/doi/10.1126/science.1188934?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed" target="_blank">link</a></span></li>
<li><span id="lu2022machine">Lu, Hongyuan, et al. "Machine learning-aided engineering of hydrolases for PET depolymerization." Nature 604.7907 (2022): 662-667. <a href="https://www.nature.com/articles/s41586-022-04599-z" target="_blank">link</a></span></li>
<li><span id="coelho2013olefin">Coelho, Pedro S., et al. "Olefin cyclopropanation via carbene transfer catalyzed by engineered cytochrome P450 enzymes." Science 339.6117 (2013): 307-310. <a href="https://www.science.org/doi/10.1126/science.1231434" target="_blank">link</a></span></li>
<li><span id="zeng2024biocatalytic">Zeng, Qing-Qing, et al. "Biocatalytic desymmetrization for synthesis of chiral enones using flavoenzymes." Nature Synthesis 3.11 (2024): 1340-1348. <a href="https://www.nature.com/articles/s44160-024-00596-4" target="_blank">link</a></span></li>
</ol>

<h3>Applications in Antibody and Other Protein Design</h3>
<ol class="bibliography">
<li><span id="jin2021refinegnn">Jin, Wengong, et al. "Iterative Refinement Graph Neural Network for Antibody Sequence-Structure Co-design." <i>ICLR 2021</i>.</span></li>
<li><span id="zhu2024abx">Zhu, Tian, Milong Ren, and Haicang Zhang. "Antibody Design Using a Score-based Diffusion Model Guided by Evolutionary, Physical and Geometric Constraints." <i>ICML 2024</i>.</span></li>
<li><span id="jin2023dsmbind">Jin, Wengong, et al. "DSMBind: SE (3) denoising score matching for unsupervised binding energy prediction and nanobody design." <i>NeurIPS 2023</i>.</span></li>
<li><span id="jin2022hern">Jin, Wengong, Regina Barzilay, and Tommi Jaakkola. "Antibody-antigen docking and design via hierarchical equivariant refinement." <i>ICML 2022</i>.</span></li>
<li><span id="divya2024rnaflow">Nori, Divya, and Wengong Jin. "Rnaflow: Rna structure & sequence design via inverse folding-based flow matching." <i>ICML 2024</i>.</span></li>
<li><span id="huang2024faformer">Huang, Tinglin, et al. "Protein-nucleic acid complex modeling with frame averaging transformer." <i>NeurIPS 2024</i>.</span></li>
</ol>

    </div>
  </div>
</div>
  </div>
  <div class="footer">
  <div class="container">
    <div class="row">
      <div class="col-12 mt-2 mb-4">
        <div class="footer-inner">
          <h3 class="footer-title">Artificial Intelligence for Protein Design, PKU Summer School 2025</h3>
        </div>
      </div>
    </div>
  </div>
</div>
  <script type="text/javascript" src="https://deepgraphlearning.github.io/ProteinTutorial_AAAI2025/assets/js/scripts.js""></script>
  
</body>
</html>
